### ISSUE TITLE
Harmonize CSS styling on metric visualization background banner border radius

### ISSUE THREAD

**Issue by netimate** [opensearch-project/OpenSearch-Dashboards#1644]
The colored background of the metric visualization currently has no border radius and thus stands out with the rounding of the metric visualization itself.

**Describe the solution you'd like**

The background of the value of the metric visualization should have the same border radius as the border of the metric visualization itself (~4px). This results in a much calmer overall picture and the visualization is calmer and more pleasant to look at.

**Describe alternatives you've considered**

Add a border radius of 4px to CSS class .mtrVis__container should harmonize the styling improving the user experience (quick win).

** FROM - Current CSS settings **
.mtrVis__container {
    text-align: center;
    padding: 16px;
}

** TO - Adapted CSS settings **
.mtrVis__container {
    text-align: center;
    padding: 16px;
    border-radius: 4px;
}

**Additional context**
With new harmonized border radius:
<img width="336" alt="metric_vis_css_with_border_radius" src="https://user-images.githubusercontent.com/106587230/171189764-2b780917-d53f-4cb2-9f87-d08a399e91f1.png">

Current styling with different border radius (missing border radius inside at the green background at the text):
<img width="335" alt="metric_vis_css_no_border_radius" src="https://user-images.githubusercontent.com/106587230/171189777-23699721-4095-4057-8612-325cb00a4efb.png">

I would like to thank you again for your commitment and the great product and I hope I was able to contribute at least a small part and give you some support.

All the best
Patrick

**Comment by ahopp** [comment]
@btzeng @kgcreative Any thoughts here? It seems like a fine update to me, but I don't know what the broader design guidance has been.

**Comment by ashwin-pc** [comment]
I like this suggestion, but just rounding them out might be a little troublesome. We can have more than one metric that is visualized in a metric visualization. e.g. 
![image](https://user-images.githubusercontent.com/20453492/174897520-f2b594c6-d189-4bce-b56f-723ea1c9ed20.png)

In this case, not having the border radius helps because it aligns well with the adjacent metric. That being said multiline metrics look terrible when we have a lot of vertical space. 
![image](https://user-images.githubusercontent.com/20453492/174897810-2b664959-d2c7-4a0c-8a60-85c3d8e0d659.png)

What i think might be better here would be to have a toggle in the style section that can enable border radius for metric visualizations that can also add some space between multiple metrics when the borders are rounded off. Basically, i support having a toggle to round off the borders that also support multiple metrics well. @kgcreative @btzeng @KrooshalUX would love your thoughts on this.

**Comment by kgcreative** [comment]
thoughts on leveraging first-child / last-child in order to selectively add left/right border radii? This would mean that a single metric would be rounded, but multiple metrics would have sharp edges, with a rounded first/last.

**Comment by joshuarrrr** [comment]
Wouldn't first/last child rounded corners still look weird if we wrap to multi line? I think I lean more towards @ashwin-pc's thought that you can't really have concatenated metrics with rounded corners - you'd need to space them out.

**Comment by ahopp** [comment]
While I like the functionality og @ashwin-pc suggestion (I'm all in for configuration options) but it feels odd that a user would have the need to toggle rounding on/off on a case-by-case basis because the rounding is in issue in some case (e.g., wrapping to additional lines). 

It's likely to be more complex, but is there anyway we can solve to the multiple line issues at the same time? For example, experiences like Grafana have side stepped by spacing the metrics e.g.,

![image](https://user-images.githubusercontent.com/9532728/175110071-58172a4d-88df-4429-b413-a145ddbd27e3.png)

And by cleaning up spacing in wrapped cases e.g.;

![image](https://user-images.githubusercontent.com/9532728/175110316-2edcc6f7-e279-473a-8a8a-8a83cde3d33c.png)

And finally, it adjusts to panel sizing e.g. the same metrics as above can be resized and it still looks good;

![image](https://user-images.githubusercontent.com/9532728/175110439-2eb756d0-a197-46a3-a0cd-d3f758f2ccc9.png)
![image](https://user-images.githubusercontent.com/9532728/175110607-7e5f990f-57d0-4cc0-af56-c9af04b14636.png)

**Comment by joshuarrrr** [comment]
Yeah, that's what I meant - that we should add spacing in general. Auto-sizing would be another huge improvement - the current approach of a user-specified fixed font size + truncation does not feel intentional or particularly user friendly.

**Comment by ashwin-pc** [comment]
@ahopp I love the examples you shared. I think rather than enabling the option to just round the corners for a metric viz, this is a good time as any to improve the metric viz overall to look more polished.
